string command
float32[] values
---
int16 ack
